src/bullet_utils/env.py

```diff
@@ -117,3 +115,16 @@ def __init__(self, server=pybullet.GUI, dt=0.001):
             package_dir / "resources" / "plane_with_restitution.urdf"
         )
         self.add_object_from_urdf(plane_urdf)
+
+    """Sets friction coefficients of the env. floor
+     Args:
+        lateral (float, optional): The lateral friction coefficient of the env. floor
+        spinning (float, optional): The spinning friction coefficient of the env. floor
+        rolling (float, optional): The rolling friction coefficient of the env. floor
+    """
+    def set_floor_frictions(self, lateral=1.0, spinning=-1.0, rolling=-1.0):
```

Comment:
What's the meaning of negative spinning and rolling friction? Should they be zero by default?

Reply:
you are right, I mixed that with default values with the contact damping and stiffness. Just pushed a commit to fix that, thanks.